hack/update-go-module-dependencies.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -ex
```

Comment:
This script looks terrible :/

Reply:
I like to think that it's not pretending to be something that it's not.